lib/std/heap.zig

```diff
@@ -33,6 +33,10 @@ pub const max_page_size: usize = switch (builtin.os.tag) {
         .wasm32, .wasm64 => 64 << 10,
         else => missing_max_page_size,
     },
+    .uefi => switch (builtin.cpu.arch) {
+        .x86, .x86_64 => 4 << 10,
```

Comment:
It seems that this is 4K for all architectures: https://github.com/tianocore/edk2/blob/b158dad150bf02879668f72ce306445250838201/MdePkg/Include/Uefi/UefiBaseType.h#L180-L187